.circleci/config.yml

```diff
@@ -144,6 +149,17 @@ commands:
           key: kedro-deps-v1-win-{{ checksum "requirements.txt" }}-{{ checksum "test_requirements.txt" }}
       - win_setup_requirements
 
+  configure_ssh_keys:
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "a0:3f:6a:d9:c3:c7:c2:56:e7:38:bc:ce:d2:dc:9e:be"
```

Comment:
I don't understand how, if at all, this is used. As part of the old `clone_repo` step there was something that did `ssh-add ~/.ssh/id_rsa_47...` for the other key, but I'm pretty sure that one was for `private-kedro`. Now we don't have `clone_repo` we don't have anything that does `ssh-add` any more. 

Any help very welcome since I don't understand SSH at all 😅

Reply:
I thought the keys were used to also create a release tag in GitHub? Or is that just the `GITHUB_TAGGING_TOKEN`?

Reply:
I _think_ we only need `GITHUB_TAGGING_TOKEN` but will check this with Ivan.

Reply:
Confirmed with Ivan that we shouldn't need this SSH stuff at all any more, so I've removed all of it.